app/src/main/java/org/oppia/app/home/RouteToTopicListener.kt

```diff
@@ -0,0 +1,6 @@
+package org.oppia.app.home
+
+/** Listener for when an activity should route to a exploration. */
+interface RouteToTopicListener {
+  fun routeToTopic(explorationId: String)
```

Comment:
Change `explorationId` to `topicId` and also change the kDoc.

Reply:
Resolved